src/components/Header/Menu.tsx

```diff
@@ -27,7 +28,8 @@ const HeaderMenu: FunctionComponent = () => {
 		<div className="header__main-nav">
 			<ul className="header__menu">
 				<li>
-					<Link to="/daos">DAOs</Link>
+					<CreateGnosisSafeModal afterCreate={() => undefined} />
+					{/* <Link to="/daos">DAOs</Link> */}
```

Comment:
I believe this link should be here, at least accordingly to design. Please, add it back